.github/workflows/ci-build.yml

```
@@ -33,20 +33,38 @@
-      run: dotnet test "${{ env.solution-path }}" --no-build --configuration Release --collect "XPlat Code Coverage" -- DataCollectionRunSettings.DataCollectors.DataCollector.Configuration.Format=opencover --logger "trx;LogFileName=unittests.trx"
-
-    - name: Push Coverage to Codacy
-      shell: bash
+    - name: Build and analyze
       env:
-        CODACY_ORGANIZATION_PROVIDER: ${{ secrets.CODACY_ORGANIZATION_PROVIDER }}
-        CODACY_USERNAME: ${{ secrets.CODACY_USERNAME }}
-        CODACY_PROJECT_NAME: ${{ secrets.CODACY_PROJECT_NAME }}
-        CODACY_API_TOKEN: ${{ secrets.CODACY_API_TOKEN }}
+        SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+      shell: powershell
```

actionlint (1.7.7)

shell name "powershell" is invalid on macOS or Linux. available names are "bash", "pwsh", "python", "sh"

(shell-name)